src/main/kotlin/no/nav/k9/vaktmester/river/RiverUtils.kt

```diff
@@ -47,3 +50,17 @@ internal fun List<Arkiv>.doIfEmpty(task: () -> Unit) = when (isEmpty()) {
     else -> {}
 }
 
+internal fun ObjectNode.fieldNamesList(): List<String> = this.fieldNames().asSequence().toList()
+
+internal fun InFlight.uløstBehov(): String? {
+    val meldingsinformasjon = JsonMessage(behovssekvens, MessageProblems(behovssekvens))
+        .vaktmesterOppgave()
+        .meldingsinformasjon()
+    val løsninger = meldingsinformasjon.løsninger.fieldNamesList()
+
+    return meldingsinformasjon.behovsrekkefølge.find { behov ->
```

Comment:
hmm, blir dette det samme som å si

`return meldingsinformasjon.behovsrekkefølge.firstOrNull { it !in løsninger  }` ? 

Føler det er litt enklere å lese  i tilfelle?